external-reports/src/main/resources/org/sonar/l10n/java/rules/pmd/rules.json

```diff
@@ -0,0 +1,2306 @@
+[
+  {
+    "key": "AbstractClassWithoutAbstractMethod",
+    "name": "AbstractClassWithoutAbstractMethod",
```

Comment:
it's a not a nice name. For some other linter we transformed key, e.g. `Abstract class without abstract method`

Reply:
Changed in b56ab44. And it seems impossible to fully automate because of a few special cases...